python/oneflow/nn/graph/graph_config.py

```diff
@@ -118,6 +110,33 @@ def set_zero_redundancy_optimizer_mode(self, mode: str = "distributed_split"):
         assert mode in ("distributed_split", "non_distributed")
         self.proto.set_optimizer_placement_optimization_mode(mode)
 
+    def enable_auto_parallel(self, mode: bool = True):
+        """If true, then graph will use auto parallel algorithm.
+
+        Args:
+            mode (bool, optional): [description]. Default is True.
+        """
+        self.proto.set_enable_auto_parallel(mode)
+
+    def set_auto_parallel_computation_cost_ratio(self, ratio):
+        """
+        Set coefficient of computation cost in auto-parallel algorithm.
+        """
+        self.proto.set_auto_parallel_computation_cost_ratio(ratio)
+
+    def set_auto_parallel_wait_time(self, cost):
+        """
+        TODO
+        """
+        self.proto.set_auto_parallel_wait_time(cost)
+
+    def set_auto_parallel_transfer_cost(self, cost):
+        """
+        TODO
+        """
+        self.proto.set_auto_parallel_transfer_cost(cost)
```

Comment:
这个注释你来描述一下我加上吧 @Yipeng1994

Reply:
wait time: An auto-parallel parameter. Describe the mutable extra time it will take when communication between devices occurs. It will be added to the copy cost and may get reduced when cover by computation cost.

transfer cost: An auto-parallel parameter. Describe the fixed extra time it will take when communication between devices occurs. It will be added to the copy cost and can not be reduced.